docs/versioned/install/README.md

```diff
@@ -8,58 +8,132 @@ function: reference
 
 # Installing Knative
 
-There are three installation methods to install Knative:
+This page provides guidance for Kubernetes administrators on how to install Knative on an existing Kubernetes cluster. Knative has three components: Eventing, Serving, and Functions along with the CLI client tools `kn` and `func`. Serving and Eventing are installed into clusters. Functions is not installed into clusters given that the client `func` CLI tool builds and deploys stateless functions as standard containers.
```

Comment:
```suggestion
This page provides guidance for Kubernetes administrators on how to install Knative on an existing Kubernetes cluster. Knative has three components: Eventing, Serving, and Functions. Serving and Eventing are installed into clusters. Functions is not installed into clusters given that the client `func` CLI tool builds and deploys stateless functions as standard containers.
```

I don't think this page needs to get into the various client installation options; Functions and `func` are also the same, so you sort of duplicated the text when adding it as a CLI tool as well as a component.